// -*- mode: c++; c-file-style: "k&r"; c-basic-offset: 4 -*-
// vim: set ts=4 sw=4:
/***********************************************************************
 *
 * qwstore/replica.h:
 *   Runs a single ORStore server replica
 *
 **********************************************************************/

#ifndef _QW_REPLICA_H_
#define _QW_REPLICA_H_

#include "paxos-lib/lib/configuration.h"
#include "paxos-lib/lib/udptransport.h"
#include "qwstore/server.h"
#include "qwstore/qwstore.h"

#endif /* _QW_REPLICA_H_ */
